std_msgs/Header header

uint32 stationId

float64 SpeedLimit

float64 dis2inter

uint8 Str_is_val
uint8 StrSignalState
float64 StrlikelyEndTime

uint8 Left_is_val
uint8 LeftSignalState
float64 LeftlikelyEndTime

uint8 Right_is_val
uint8 RighSignalState
float64 RightlikelyEndTime
